test/NuGet.config

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why is this here? I imagine that the tests projects should use the same RestoreSources as the source projects. Otherwise, we will end up with packages outside the build feed.